odoo_backup_sh/README.rst

```diff
@@ -34,17 +34,26 @@ Implementation details
 /web/database/backups
 ---------------------
 
-It's a new page in *database manager*, that allows to see list of available backups that can be restored. The process of receiving backup list is as following:
+It's a new page in *database manager*, that allows to see list of available backups that can be restored. Below is process of receiving backup list. Click on the ``Restore via Odoo-backup.sh`` consists the following operations:
 
-* *Browser* sends request to ``/web/database/backups``
-* *Odoo instance* sends request to ``odoo-backup.sh/web/backup/list`` with ``user_key`` from odoo config file (it's generated automatically if it's not set).
-* ``odoo-backup.sh`` **either** returns backup list and then browser shows backup list, **or** ``odoo-backup.sh`` doesn't recognise ``user_key`` and starts process of authentication via ``odoo.com`` as auth provider:
+* If there are not amazon S3 credentials in the system parameters, the module makes requests to get them:
 
-  * ``odoo-backup.sh/web/backup/list`` returns a link to redirect browser
-  * *Browser* recieves from *Odoo instance* an html page that only redirects *User* to a technical page page in ``odoo-backup.sh``, which finally redirects *User* to ``odoo.com`` website
-  * *User* enters his login-password and redirects back to a technical page in ``odoo-backup.sh`` from where he is redirected again to ``/web/database/backups`` page and the process starts again, but this time  ``odoo-backup.sh`` recognise ``user_key``
-  
+  * *Browser* sends request to ``/web/database/backups``
+  * *Odoo instance* sends request to ``odoo-backup.sh/web/backup/list`` with ``user_key`` from odoo config file (it's generated automatically if it's not set).
```

Comment:
rename url to something more appropriate
``odoo-backup.sh/web/backup/cloud``
``odoo-backup.sh/web/backup/get-cloud``
etc